src/components/main-app/ol-init.js

```diff
@@ -80,9 +113,46 @@ export default function OlInit() {
               Mapbox
             </a>
           </div>
+          <div className="button-block">
+            <button
+              className="toggle"
+              onClick={e => onChangeBaseLayer('light', URL_UPDATE_PUSH)}
+            >
+              light
+            </button>
+            <button
+              className="toggle"
+              onClick={e => onChangeBaseLayer('dark', URL_UPDATE_PUSH)}
+            >
+              dark
+            </button>
+            <button
+              className="toggle"
+              onClick={e => onChangeBaseLayer('satelight', URL_UPDATE_PUSH)}
+            >
+              satelight
+            </button>
+            <button
+              className="toggle"
+              onClick={e => onChangeBaseLayer('street', URL_UPDATE_PUSH)}
+            >
+              street
+            </button>
+            <button
+              className="toggle"
+              onClick={e => onChangeBaseLayer('green', URL_UPDATE_PUSH)}
+            >
+              green
+            </button>
```

Comment:
Can have a single component for this and dynamically generate with help of a map function